jolie/src/main/java/jolie/Interpreter.java

```diff
@@ -272,7 +277,7 @@ public CorrelationSet correlationInitializer() {
 	private final long persistentConnectionTimeout = 60 * 60 * 1000; // 1 hour
 	private final long awaitTerminationTimeout = 60 * 1000; // 1 minute
 
-
+	private Map< URI, SymbolTable > symbolTables;
```

Comment:
This field is necessary in the future commit, when we enable the mechanism to embed a service node. The implementation was extended from embedded internal service where the loader initiate new interpreter with the current interpreter and parsed AST program. In service node version, the created interpreter accesses and passes parent's symboltable to semantic verification phase. Otherwise the verification phase in embed program would have no clue if the symbol has defined.

Reply:
This field will be moved to InterpreterParameter